src/FSharpSpreadsheetML/SheetData.fs

```diff
@@ -18,9 +18,14 @@ module SheetData =
         sheetData.AppendChild row |> ignore
         sheetData
 
+    //----------------------------------------------------------------------------------------------------------------------
+    //                                              Row(s)                                                                  
+    //----------------------------------------------------------------------------------------------------------------------
+
     /// Returns a sequence of row contained in the sheetdata
     let getRows (sheetData:SheetData) = 
         sheetData.Descendants<Row>()
```

Comment:
Is this additional cast really necessary?

Reply:
Readability improving. But ill add a type annotation instead.